content/_index.md

```diff
@@ -10,15 +10,7 @@ title="Home"
 	}
 </style>
 
-## Welcome
-
-<div class="images">
-	<a href="/about-us">
-		{{ resize_image(path="IMG_2480.JPEG", width=1920, height=1080, op="fit", format="auto", quality=90) }}
-	</a>
-</div>
-
-### Join us!
+## Join us!
```

Comment:
`Join Zoom Meeting` below should be promoted to a third-level header with this change.